docs/metrics.rst

```diff
@@ -75,13 +77,16 @@ pool.starving_tasks.<pool_name>                 Number of starving tasks in the
 Timers
 ------
 
-================================= =================================================
-Name                              Description
-================================= =================================================
-dagrun.dependency-check.<dag_id>  Seconds taken to check DAG dependencies
-dag.<dag_id>.<task_id>.duration   Seconds taken to finish a task
-dagrun.duration.success.<dag_id>  Seconds taken for a DagRun to reach success state
-dagrun.duration.failed.<dag_id>   Seconds taken for a DagRun to reach failed state
-dagrun.schedule_delay.<dag_id>    Seconds of delay between the scheduled DagRun
-                                  start date and the actual DagRun start date
-================================= =================================================
+=========================================== =================================================
+Name                                        Description
+=========================================== =================================================
+dagrun.dependency-check.<dag_id>            Seconds taken to check DAG dependencies
+dag.<dag_id>.<task_id>.duration             Seconds taken to finish a task
+dagrun.duration.success.<dag_id>            Seconds taken for a DagRun to reach success state
+dagrun.duration.failed.<dag_id>             Seconds taken for a DagRun to reach failed state
+dagrun.schedule_delay.<dag_id>              Seconds of delay between the scheduled DagRun
+                                            start date and the actual DagRun start date
+webserver.<endpoint>.load_time              Time it takes to load server side data
+webserver.tree.load_time                    Time it takes to render a DAG in tree view
```

Comment:
Also `load server side data` here as we don't include rendering.

Reply:
Fixed.